curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/665076818092e53529c2364d.md

```diff
@@ -0,0 +1,76 @@
+---
+id: 665076818092e53529c2364d
+title: Step 21
+challengeType: 1
+dashedName: step-21
+---
+
+# --description--
+
+In the last few steps, you learned all about working with arrays. Now it is time to review what you have learned so you can apply this knowledge to the pyramid generator project.
+
+In this step, you will need to do the following:
+
+- Create a variable called `cities` and assign it an array of three strings: `"London"`, `"New York"`, `"Mumbai"`
+- On a new line, log the entire array to the console
+- On a new line, update the last element of the array to the string `"Mexico City"`
+- On a new line, log the entire array to the console
+
+When done correctly, you should see this output in the console
+
+```js
+[ "London", "New York", "Mumbai" ]
+[ "London", "New York", "Mexico City" ]
+```
+
+# --hints--
+
+You should use ```let``` to create a ```cities``` variable.
```

Comment:
```suggestion
You should use `let` to create a `cities` variable.
```

Inline code should only use one backtick, not three. This will need to be corrected throughout your changes.